src/utils.test.js

```diff
@@ -214,6 +214,12 @@ describe('utils', () => {
     expect(utils.isStoreLogs()).toBe(true);
   });
 
+  test('isReuseHttpConnection', () => {
+    expect(utils.isReuseHttpConnection()).toBe(false);
+    process.env['LUMIGO_REUSE_HTTP_CONNECTION'] = 'TRUE';
+    expect(utils.isReuseHttpConnection()).toBe(true);
+  });
+
   test('setSendOnlyIfErrors', () => {
     expect(utils.isSendOnlyIfErrors()).toBe(false);
     utils.setSendOnlyIfErrors();
```

Comment:
Another issue, if the lambda tries to reuse the connection but the edge closed the connection. What will happen? Will it recreate the connection or the sending of the span will fail?
Unittest: Try to mock closed connection (see how hard it is, it might be too complicated).
Manual test or IT: maybe run a stress test to see that spans are still sent.

Reply:
Yes, it should re-create it. I'm trying to add a test as u suggested